apps/files/src/components/FileList.vue

```diff
@@ -128,7 +128,7 @@ export default {
     Mixins
   ],
   name: 'FileList',
-  props: ['fileData', 'starsEnabled', 'checkboxEnabled', 'dateEnabled'],
+  props: ['fileData', 'starsEnabled', 'checkboxEnabled', 'dateEnabled', 'parentFolder'],
```

Comment:
Why using prop instead of getter from state directly in component?

Reply:
Just following the existing patter - files are pushed in via props - just doing it the same way for the parent folder.

I didn't want to start a refactor now to get rid of all the props :man_shrugging:

Reply:
I see. Thanks for info